frontend/src/lib/requests.ts

```diff
@@ -18,17 +20,20 @@ export type ApiResponse =
     | { [x: string]: ApiResponse };
 
 export async function makeRequest<TData extends ApiResponse>(
-    config: AxiosRequestConfig
+    requestConfig: AxiosRequestConfig
 ): Promise<TData> {
     try {
-        const response = await quizootApi(config);
+        if (requestConfig.method != 'GET') {
```

Comment:
Hummmm.. I'm not convinced by this one. We should do it manually instead of making it generic.

I prefer `makeRequest({url: \`/api/quizzes\`, method: 'GET' })` and `makeRequest({url: \`/api/quizzes/\`, method: 'POST' , data: {} })`

With your approach, what would happen if I do :
```typescript
makeRequest({
  url: '/api/quizzes/',
  method: 'POST'
}}
```

You could still find a way to make it generic, but let's not do it. It can lead to unexpected behavior

Reply:
What I was rightly trying to do was to avoid writing `/api/quizzes/` in one place and `/api/quizzes` in another.
But yeah, we can go without it

Reply:
Yeah I know. But I really think that we might endup quickly with `//`. Anyway, the developer should test every request so if `/api/quizzes` doesn't work, the we can try `/api/quizzes/`

Reply:
Yeah, you got a point